lib/question.js

```diff
@@ -583,6 +598,18 @@ module.exports = {
    * @param {function} callback - A callback(err, submission_id) function.
    */
   saveSubmission(submission, variant, question, course, callback) {
+    // This is (hopefully) temporary code to debug an issue where chunk servers
+    // get stuck inside of a transaction. We can possibly remove it once we
+    // identify and fix the issue.
+    const timeoutId = setTimeout(() => {
+      logger.error('saveSubmission transaction was not closed within 60 seconds', {
```

Comment:
Might be worth including some information about the submission, in case it's bad submissions causing this? Could even just be something like `JSON.stringify(submission).substring(0, 1000)` so that we truncate it if it's too big?

Reply:
I like your thinking, I made this change in 4de3e5e.